scanner/src/funTest/assets/dummy-expected-scan-results-for-analyzer-result.yml

```diff
@@ -0,0 +1,132 @@
+---
```

Comment:
Commit message nits:
- `OrtResults` should probably be `OrtResult`s or `OrtResult`'s (as in the next commit)
- Not sure about the comma before "by"